vocabularies/PersonalData.xml

```diff
@@ -142,9 +142,8 @@ Terms for contact and address information are defined in the [Communication voca
               <PropertyValue Property="Value" String="PurposeID" />
               <Annotation Term="Core.Description" String="The unique identifier for the purpose of a consent" />
               <Annotation Term="Core.LongDescription">
-                <String>The purpose of a consent is the information that specifies the reason and the goal for
-                the processing of a specific set of personal data. As a rule, the purpose
-                references the relevant legal basis for the processing of personal data.</String>
+                <String>The unique identifier for the purpose of processing of personal data. Any processing of personal data is based on specified, explicit, and legitimate purposes and not further processed in a manner that is incompatible with those purposes. The purposes are defined by the data controller or joint data controllers.
```

Comment:
```suggestion
                <String>Any processing of personal data is based on specified, explicit, and legitimate purposes, and data are not further processed in a manner that is incompatible with those purposes. The purposes are defined by the data controller or joint data controllers.
```
- Do not repeat the Description in the LongDescription.
- I split the sentence in two (separated by a new comma), the previous wording was unclear to me.
- Clarify the role of the consent. I assume that any given purpose must be based on either a legitimate interest of on the consent of the data owner.